src/taxonomy/tag-list/data/api.js

```diff
@@ -9,8 +9,8 @@ import { camelCaseObject, getConfig } from '@edx/frontend-platform';
 import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
 
 const getApiBaseUrl = () => getConfig().STUDIO_BASE_URL;
-const getTagListApiUrl = (taxonomyId, page) => new URL(
-  `api/content_tagging/v1/taxonomies/${taxonomyId}/tags/?page=${page + 1}`,
+const getTagListApiUrl = (taxonomyId, page, pageSize) => new URL(
+  `api/content_tagging/v1/taxonomies/${taxonomyId}/tags/?page=${page + 1}&page_size=${pageSize}`,
   getApiBaseUrl(),
 ).href;
```

Comment:
Just a suggestion that I think might help avoid any issues that would be hard to spot when needing to update/add queryparams in the future.

```suggestion
const getTagListApiUrl = (taxonomyId, page, pageSize) => {
  const url = new URL(`api/content_tagging/v1/taxonomies/${taxonomyId}/tags/`, getApiBaseUrl());
  url.searchParams.append('page', page + 1);
  url.searchParams.append('page_size', pageSize);
  return url.href;
};
```

Reply:
Thanks!